project/CopyrightHeader.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka
+
+import sbt._, Keys._
+import de.heikoseeberger.sbtheader.HeaderPlugin
+
+object CopyrightHeader extends AutoPlugin {
+  import HeaderPlugin.autoImport.{ HeaderLicense, headerCheck, headerCreate, headerLicense }
+  import ValidatePullRequest.{ additionalTasks, ValidatePR }
+
+  override def requires = HeaderPlugin
+  override def trigger = allRequirements
+
+  override def projectSettings = Def.settings(
+    Seq(Compile, Test).flatMap { config =>
+      inConfig(config)(
+        Seq(
+          includeFilter in headerCreate := (includeFilter in unmanagedSources).value,
+          excludeFilter in headerCreate := (excludeFilter in unmanagedSources).value,
+          unmanagedSources in headerCreate := Defaults.collectFiles(
```

Comment:
@hseeberger (Sorry to ping you twice) As mentioned in https://github.com/sbt/sbt-header/pull/147 I could not get `excludeFilter.in(unmanagedSources.in(headerCreate))` to work. It could be the lack of configuration scope however, the [sbt-header code](https://github.com/sbt/sbt-header/blob/fa041e5ca0f5c1ac4ca5c558abfb1384a568b4fc/src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala#L95) doesn't seem to use `Defaults.collectFiles` or something similar to apply these filters. Is this a bug?

Reply:
Probably a Bug/missing feature.